docs/configuration/access-control.md

```diff
@@ -8,176 +8,395 @@ nav_order: 1
+expressed in different ways.*
 
-### Domains
+```yaml
+access_control:
+  rules:
+  - domain: example.com
+    policy: one_factor
+    subject: "group:super-admin"
+  - domain: example.com
+    policy: one_factor
+    subject: 
+    - "group:super-admin"
+  - domain: example.com
+    policy: one_factor
+    subject:
+    - ["group:super-admin"]
+```
+
+### methods
+<div markdown="1">
+type: list(string)
+{: .label .label-config .label-purple } 
+required: no
+{: .label .label-config .label-green }
+</div>
+
+This criteria matches the HTTP request method. This is primarily useful when trying to bypass authentication for specific
+request types when those requests would prevent essential or public operation of the website. An example is when you
+need to do CORS preflight requests you could apply the `bypass` policy to `OPTIONS` requests.
+
+It's important to note that Authelia cannot preserve request data when redirecting the user. For example if the user had
+permission to do GET requests, their authentication level was `one_factor`, and POST requests required them to do
+`two_factor` authentication, they would lose the form data. Additionally it is sometimes not possible to redirect users
+who have done requests other than HEAD or GET which means the user experience may suffer. These are the reasons it's
+only recommended to use this to increase security where essential and for CORS preflight.
+
+Example:
+
+```yaml
+access_control:
+  rules:
+  - domain: example.com
+    policy: bypass
+    methods:
+    - OPTIONS
+```
 
-The domains defined in rules must obviously be either a subdomain of the domain
-protected by Authelia or the protected domain itself. In order to match multiple
-subdomains, the wildcard matcher character `*.` can be used as prefix of the domain.
-For instance, to define a rule for all subdomains of *example.com*, one would use
-`*.example.com` in the rule. A single rule can define multiple domains for matching.
-These domains can be either listed in YAML-short form `["example1.com", "example2.com"]`
-or in YAML long-form as dashed list.
+The valid request methods are: OPTIONS, HEAD, GET, POST, PUT, PATCH, DELETE, TRACE, CONNECT. Additional information 
+about HTTP request methods can be found on the [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).
 
-Domain prefixes can also be dynamically match users or groups. For example you can have a 
-specific policy adjustment if the user or group matches the subdomain. For
-example `{user}.example.com` or `{group}.example.com` check the users name or 
-groups against the subdomain.
+### networks
+<div markdown="1">
+type: list(string)
+{: .label .label-config .label-purple } 
+required: no
+{: .label .label-config .label-green }
+</div>
 
-### Resources
+This criteria is a list of network address ranges in CIDR notation or an alias from the [global](#networks-global)
```

Comment:
The network addresses do not need to be in CIDR notation, they can be singlular IP addresses or CIDR notation.